frontend.md

```diff
@@ -0,0 +1,190 @@
+
+<a name="php-starter-kits"></a>
+### Starter Kits
+
+If you would like to build your frontend using PHP and Livewire, you can leverage our Breeze or Jetstream [starter kits](/docs/{{version}}/starter-kits) to jump-start your application's development. Both of these starter kits scaffold application's backend and frontend authentication flow using [Blade](/docs/{{version}}/blade) and [Tailwind](https://tailwindcss.com) so that you can start building your next big idea.
+
+<a name="using-vue-react"></a>
+## Using Vue / React
+
+Although it's possible to build modern frontends using Laravel and Livewire, many developers still prefer to leverage the power of a JavaScript framework like Vue or React. This allows developers to take advantage of the rich ecosystem of JavaScript packages and tools available via NPM.
+
+However, without additional tooling, pairing Laravel with Vue or React would leave us needing to solve a variety of complicated problems: client-side routing, data hydration, and authentication. Client-side routing is often simplified by using opinionated Vue / React frameworks such as [Nuxt]() and [Next](); however, data hydration and authentication remain complicated and cumbersome problems to solve when pairing a backend framework like Laravel with these frontend frameworks.
+
+In addition, developers are left maintaining two separate code repositories, often needing to coordinate maintenance, releases, and deployments across both repositories. While these problems are not insurmountable, we don't believe it's a productive or enjoyable way to develop applications.
+
+<a name="inertia"></a>
+### Inertia
+
+Thankfully, Laravel has an answer to this problem. [Inertia](https://inertiajs.com) bridges the gap between your Laravel application and your modern Vue or React frontend, allowing you to build full-fledged, modern frontends using these frameworks while leveraging Laravel routes and controllers to handle routing, data hydration, and authentication — all within a single code repository.
+
+After installing Inertia into your Laravel application, you will write routes and controllers like normal. However, instead of returning a Blade template from your controller, you will return an Inertia page:
+
+    <?php
+
+    namespace App\Http\Controllers;
+
+    use App\Http\Controllers\Controller;
+    use App\Models\User;
+    use Inertia\Inertia;
+
+    class UserController extends Controller
+    {
+        /**
+         * Show the profile for a given user.
+         *
+         * @param  int  $id
+         * @return \Inertia\Response
+         */
+        public function show($id)
+        {
+            return Inertia::render('Users/Profile', [
+                'user' => User::findOrFail($id)
+            ]);
+        }
+    }
```

Comment:
```suggestion
```php
<?php

namespace App\Http\Controllers;

use App\Http\Controllers\Controller;
use App\Models\User;
use Inertia\Inertia;

class UserController extends Controller
{
    /**
     * Show the profile for a given user.
     *
     * @param  int  $id
     * @return \Inertia\Response
     */
    public function show($id)
    {
        return Inertia::render('Users/Profile', [
            'user' => User::findOrFail($id)
        ]);
    }
}
```
```